makefile

```diff
@@ -125,6 +125,7 @@ clean-deps:
 	@$(RM) $(call findfiles,test,*.d.*)
 	@$(RM) $(call findfiles,lib,*.d.*)
 	@$(RM) $(call findfiles,stan,*.dSYM)
+	@$(RM) $(call findfiles,make,ucrt)
```

Comment:
Just noting so we remember to do it: it seems like stan and cmdstan do _not_ call the clean target of their submodules, so we will need to manually add something which deletes this higher up.